src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -34,45 +37,33 @@ public List<Publication> List()
-            // TODO a user is connected to Publications for the purpose of administration. Once this has been modelled
-            // TODO then this method will need altered reflect this.
-            return _context.Publications.Select(p => new Publication
-                {
-                    Id = p.Id,
-                    Title = p.Title,
-                    NextUpdate = p.NextUpdate, // Null | Partial Date | Complete Date
-                    Contact = p.Contact != null ? new Contact
-                    {
-                        Id = p.Contact.Id,
-                        ContactName = p.Contact.ContactName,
-                        TeamEmail = p.Contact.TeamEmail,
-                        ContactTelNo = p.Contact.ContactTelNo
-                    } : null,
-                    TopicId = p.TopicId,
-                    Methodologies = p.Methodologies.Select(m => new Methodology
-                    {
-                        Id = m.Id,
-                        Title = m.Title
-                    }).ToList(),
-                    Releases = p.Releases.Select(r => new Release
-                        {
-                            Id = r.Id,
-                            Title = r.Title,
-                            Published = r.Published,
-                            ReleaseName = r.ReleaseName,
-                            TimePeriodCoverage = r.TimePeriodCoverage,
-                            // TODO isLatestRelease - This needs to be done when we have a chronological concept of 
-                            // TODO different releases. Note it can't be published date as technically there is no
-                            // TODO reason why releases have to be published in order. 
-                            // TODO LastEdited - This needs to be done when we have a concept of release versions
-                        }
-                        
-                    ).ToList()
-                }
-            ).Where(p => p.TopicId == topicId).ToList();
+            var config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<Publication, PublicationViewModel>();
+                cfg.CreateMap<Release, ReleaseViewModel>()
+                    .ForMember(
+                        dest => dest.LatestRelease, m => m.MapFrom(release => IsLatestRelease(release)));
+                cfg.CreateMap<Methodology, MethodologyViewModel>();
+            });
+
+            var mapper = config.CreateMapper();
+
+            var publications = _context.Publications.Where(p => p.TopicId == topicId)
+                .Include(p => p.Contact)
+                .Include(p => p.Releases)
+                .Include(p => p.Methodologies)
+                .ToList();
+            
+            return mapper.Map<List<PublicationViewModel>>(publications);
+        }
+        
+        private bool IsLatestRelease(Release release)
+        {
+            return !release.Publication.Releases.Exists(
```

Comment:
Is there any advantage of sticking this on the Release object itself as a transient field?